Documentation/helm-ceph-cluster.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Helm Ceph Cluster Chart
```

Comment:
This will show up in the table of contents under "Ceph Operator", so "Ceph Cluster" seems more consistent
```suggestion
title: Ceph Cluster
```